### PR TITLE
Align C4 models

### PR DESCRIPTION
**Pull request by dstenroejl**
- [x] Use new C4 model structure
- [x] Align folder structure

Part of:
https://app.zenhub.com/workspaces/the-outlaws-6193fe815d79fc0011e741b1/issues/gh/energinet-datahub/team-the-outlaws/1100

### CODE REVIEW

**Review by kft — DISMISSED** [review]

File: docs/diagrams/c4-model/views.dsl [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        #
+        !ref dh3 {
+
+            # IMPORTANT:
+            # The order by which models are included is important for how the domain-to-domain relationships are specified.
+            # A domain-to-domain relationship should be specified in the "client" of a "client->server" dependency, and
+            # hence domains that doesn't depend on others, should be listed first.
+
+            # Include Market Participant model
+            !include https://raw.githubusercontent.com/Energinet-DataHub/geh-market-participant/dstenroejl/align-c4-models/docs/diagrams/c4-model/model.dsl
```
Comment (dstenroejl) [inline docs/diagrams/c4-model/views.dsl:25]: Use main branch

File: docs/diagrams/c4-model/views.dsl [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+            # Include EDI model
+            !include https://raw.githubusercontent.com/Energinet-DataHub/opengeh-edi/dstenroejl/align-c4-models/docs/diagrams/c4-model/model.dsl
+
+            # Include Wholesale model
+            !include https://raw.githubusercontent.com/Energinet-DataHub/opengeh-wholesale/dstenroejl/align-c4-models/docs/diagrams/c4-model/model.dsl
```
Comment (dstenroejl) [inline docs/diagrams/c4-model/views.dsl:31]: Use main branch

File: docs/diagrams/c4-model/views.dsl [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+            # Include Frontend model
+            !include https://raw.githubusercontent.com/Energinet-DataHub/greenforce-frontend/dstenroejl/align-c4-models/docs/diagrams/c4-model/model.dsl
```
Comment (dstenroejl) [inline docs/diagrams/c4-model/views.dsl:34]: Use main branch

File: docs/diagrams/c4-model/views.dsl (hunk 2 above)
Comment (dstenroejl) [reply docs/diagrams/c4-model/views.dsl:30]: Not possible at the moment

File: docs/diagrams/c4-model/views.dsl (hunk 3 above)
Comment (dstenroejl) [reply docs/diagrams/c4-model/views.dsl:33]: Not possible at the moment

**Review by djorgensendk — APPROVED** [review]

**Review by MadsDue — APPROVED** [review]

**Review by MadsDue — APPROVED** [review]

**Pull request merged**